tests/NHSD.BuyingCatalogue.Identity.Api.IntegrationTests/Utils/Settings.cs

```diff
@@ -10,30 +10,24 @@ public sealed class Settings
 
         public string OrganisationApiBaseUrl { get; }
 
-        public SmtpServer Smtp { get; }
+        public SmtpServerSettings Smtp { get; }
 
         public Settings(IConfigurationRoot config)
         {
             AdminConnectionString = config.GetConnectionString("CatalogueUsersAdmin");
             ConnectionString = config.GetConnectionString("CatalogueUsers");
             OrganisationApiBaseUrl = config.GetValue<string>("OrganisationApiBaseUrl");
-            Smtp = new SmtpServer(config.GetSection("SmtpServer"));
+            Smtp = config.GetSection("SmtpServer").Get<SmtpServerSettings>();
         }
 
-        public sealed class SmtpServer
+        public sealed class SmtpServerSettings
         {
-            public SmtpServer(IConfigurationSection config)
-            {
-                Host = config.GetValue<string>("Host");
-                Port = config.GetValue<ushort>("Port");
-                ApiBaseUrl = config.GetValue<string>("ApiBaseUrl");
-            }
 
-            public string Host { get; }
+            public string Host { get; set; }
```

Comment:
Is this used anywhere?

Reply:
Great catch, thank you. 

This is a remnant of previous implementation, which was unintentionally left and forgotten, I'll clean this mess up.